outdated-pkgs

```diff
@@ -0,0 +1,290 @@
+#!/usr/bin/python
+
+"""
+Usage:
+  For goc packages:
+      %s goc-osg  [3.3|3.4]     [el6|el7] [development|testing|release] [itb|production]
+  For osg epel packages:
+      %s osg-epel [3.4|3.5|3.5] [el6|el7] [development|testing|release]
+    Or:
+      %s osg-epel [devops]      [el7|el8] [itb|production]
+  For osg i2 packages:
+      %s osg-i2   [3.3|3.4]     [el6|el7] [development|testing|release]
+
+Default for goc:
```

Comment:
With the combined script, the usage summary is less obvious what the different modes are actually doing.  I feel like a few more words can help to clarify:

```suggestion
  For goc packages behind osg versions:
      %s goc-osg  [3.3|3.4]     [el6|el7] [development|testing|release] [itb|production]
  For osg packages behind epel versions:
      %s osg-epel [3.4|3.5|3.5] [el6|el7] [development|testing|release]
    Or:
      %s osg-epel [devops]      [el7|el8] [itb|production]
  For osg packages behind i2 versions:
      %s osg-i2   [3.3|3.4]     [el6|el7] [development|testing|release]

Default for goc-osg:
```